### PR TITLE
Fix PIL imports

### PR DESCRIPTION
**Pull request by akx**
## Description

Follows up on #7116, which changed some of these imports.

## 🎯 PRs Should Target Issues

See #7116.

### CODE REVIEW

**Comment by abidlabs** [comment]
Thanks @akx lgtm feel free to open for review and we can get this merged in quite quickly

File: gradio/components/annotated_image.py [hunk 1]
```diff
@@ -15,8 +15,6 @@
 
 set_documentation_group("component")
 
-PIL.Image.init()  # fixes https://github.com/gradio-app/gradio/issues/2843
```
Comment (abidlabs) [inline gradio/components/annotated_image.py:18]: We should keep this change as older versions of PIL (before 9.4.0) have the reported bug. Gradio pins to PIL 8.0 or higher

File: gradio/components/base.py [hunk 2]
```diff
@@ -34,7 +33,6 @@ class DataframeData(TypedDict):
 
 
 set_documentation_group("component")
-_Image.init()  # fixes https://github.com/gradio-app/gradio/issues/2843
```
Comment (abidlabs) [inline gradio/components/base.py:37]: Same

File: gradio/components/image.py [hunk 3]
```diff
@@ -18,7 +18,6 @@
 from gradio.events import Events
 
 set_documentation_group("component")
-PIL.Image.init()  # fixes https://github.com/gradio-app/gradio/issues/2843
```
Comment (abidlabs) [inline gradio/components/image.py:21]: Same

File: gradio/components/image_editor.py [hunk 4]
```diff
@@ -18,10 +18,9 @@
 from gradio.events import Events
 
 set_documentation_group("component")
-_Image.init()  # fixes https://github.com/gradio-app/gradio/issues/2843
```
Comment (abidlabs) [inline gradio/components/image_editor.py:21]: Same

File: gradio/image_utils.py [hunk 5]
```diff
@@ -4,20 +4,20 @@
 from typing import Literal
 
 import numpy as np
-from PIL import Image as _Image  # using _ to minimize namespace pollution
+import PIL.Image
 
 from gradio import processing_utils
 
-_Image.init()
```
Comment (abidlabs) [inline gradio/image_utils.py:11]: Same

File: gradio/components/annotated_image.py (hunk 1 above)
Comment (abidlabs) [inline gradio/components/annotated_image.py:18]: We may need to keep this one as we don't import image_utils here

**Comment by abidlabs** [comment]
Just one note above, otherwise LGTM @akx

File: gradio/components/annotated_image.py (hunk 1 above)
Comment (akx) [reply gradio/components/annotated_image.py:18]: There's no way to import `gradio.components.annotated_image` without the parent module `gradio.components` getting imported, which imports `gradio.components.image` and then `image_utils`.

**Review by abidlabs — APPROVED** [review]
LGTM

**Comment by abidlabs** [comment]
Merging, thanks again @akx

**Pull request merged**